go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/awslabs/soci-snapshotter
 go 1.20
 
 require (
-	github.com/containerd/containerd v1.7.11
+	github.com/containerd/containerd v1.7.6
```

Comment:
It's doing this again....

Can we close this PR? The only upgrade here is crypto which is being handled by #1004.